gravitee-apim-portal-webui/src/css/layout.css

```diff
@@ -93,7 +93,6 @@
   background-color: var(--gv-theme-color-dark);
   color: var(--gv-theme-color-lighter);
   height: var(--gv-theme-layout-footer-top--h);
-  padding-top: 55px;
```

Comment:
Always overload because the env variable `--gv-theme-layout--pt` is [always existing](https://github.com/gravitee-io/gravitee-api-management/blob/8a29363cc0a539fdd996731936066f6b584402de/gravitee-apim-portal-webui/src/styles.scss#L14)